aws-crt-kotlin/native/src/aws/sdk/kotlin/crt/Logging.kt

```diff
@@ -68,3 +68,7 @@ internal object Logging {
         }
     }
 }
+
+internal inline fun klog(level: LogLevel, message: String, subject: aws_log_subject_t = AWS_LS_KOTLIN_CRT_GENERAL) {
+    s_crt_kotlin_log(level.value.convert(), subject, message)
+}
```

Comment:
**Style**: `klog` is not the sort of identifier we'd typically use in Kotlin code. Is the `k` to differentiate from some other `log` method? Is there a reason for this not to be part of the `Logging` object?

Reply:
It was just a short hand for identifying that we are targeting the `AWS_LS_KOTLIN_CRT_GENERAL` subject (i.e. logging a kotlin related logging message to CRT logger as opposed to some other subject). 

I don't care what we name it, we could just shorten it to `log` if you prefer.

Reply:
I like `log`

Reply:
I _think_ `log` makes sense for the majority of our logging needs. _Will_ we be logging from **aws-crt-kotlin** to a subject other than `AWS_LS_KOTLIN_CRT_GENERAL`?